rust/element/src/cursor.rs

```diff
@@ -519,6 +535,13 @@ mod test {
         assert!(!cursor.is_boundary::<BaseMetric>());
     }
 
+    #[test]
+    fn count_between() {
+        let input = "\n \n \n";
+        let cursor = Cursor::new(&input, 0);
+        assert_eq!(3,cursor.count_between::<LinesMetric>(0, 5));
+    }
```

Comment:
I'd expand this test. Typically you want to include edge cases, such as empty strings, negative cases, _etc._